### PR TITLE
lazy loading through dom-if

### PR DESCRIPTION
**Pull request by benbenbenbenbenben**

### CODE REVIEW

File: paper-tree-node.html [hunk 1]
```diff
@@ -114,13 +114,15 @@
...
                 </template>
             </div>
-            <ul hidden$={{!data.open}}>
-                <template is="dom-repeat" items="{{data.children}}">
-                    <li>
-                        <paper-tree-node data="{{item}}" actions="{{actions}}"></paper-tree-node>
-                    </li>
-                </template>
-            </ul>
+            <template is="dom-if" if$="{{data.open}}">
```
Comment (vpusher) [inline paper-tree-node.html]: why are you doing an attribute binding here instead of a simple property binding ?

**Comment by vpusher** [comment]
I rebased your branch on the master in order to keep it in sync with the latest deliveries

File: paper-tree-node.html (hunk 1 above)
Comment (benbenbenbenbenben) [reply paper-tree-node.html]: I hadn't noticed I had, sorry

**Comment by benbenbenbenbenben** [comment]
I've changed the if attribute

**Comment by vpusher** [comment]
@benbenbenbenbenben great ! Merging. I will release the new version asap.

**Comment by vpusher** [comment]
@benbenbenbenbenben Released in [v1.0.3](/vpusher/paper-tree/releases/tag/v1.0.3)

**Pull request merged**